provisioning/roles/nginx/templates/letsencrypt.j2

```diff
@@ -0,0 +1,15 @@
+# Initial configuration for Let's Encrypt.
+# You should not use this configuration as the definitive one.
```

Comment:
We are temporarily using it to get an initial certificate from Let's Encrypt, then it's replaced by the `nginx_with_ssl` role. The rationale behind this was : it was impossible to obtain a certificate using the "final configuration".